app/src/main/java/mozilla/lockbox/extensions/Observable+.kt

```diff
@@ -15,5 +17,18 @@ fun <T : Any, U : T> Observable<T>.filterByType(clazz: Class<out U>): Observable
 }
 
 fun Observable<List<ServerPassword>>.mapToItemViewModelList(): Observable<List<ItemViewModel>> {
-    return this.map { it.map { it.toViewModel() } }
+    return this.map { list -> list.map { it.toViewModel() } }
+}
+
+fun Observable<Pair<List<ItemViewModel>, Int>>.sort(): Observable<List<ItemViewModel>> {
+    return this.map { pair ->
+        when (pair.second) {
+            R.id.sort_a_z -> { pair.first.sortedBy { it.title } }
+            R.id.sort_recent -> { pair.first.sortedBy { it.timeLastUsed }}
+            else -> {
+                log.error("Unrecognized sort option ${pair.second}")
+                pair.first.sortedBy { it.title }
+            }
+        }
```

Comment:
This is weird. This should likely go in the `ItemListPresenter`, and my using `combineLatest` (or the equivalent) of the list and the `ItemListSort`.